services/src/resolvers/buddy.ts

```diff
@@ -11,14 +11,14 @@ const newbies: ResolverFn<
   ResolversParentTypes['Buddy'],
   Context,
   any
-> = (parent, args, context) => context.prisma.buddy({ id: parent.id }).newbies();
+> = (parent, _args, context) => context.prisma.buddy({ id: parent.id }).newbies();
 
 const newbiesCount: ResolverFn<
   Buddy['newbiesCount'],
   ResolversParentTypes['Buddy'],
   Context,
   any
-> = async (parent, args, context) => {
+> = async (parent, _args, context) => {
```

Comment:
Just curious, why it's needed?

Reply:
It was automatic conversion from vscode. 
<img width="546" alt="Screenshot 2020-01-24 at 12 51 50" src="https://user-images.githubusercontent.com/12997200/73067381-d1974100-3ea8-11ea-81db-4f42b0e683b6.png">

Reply:
Right... maybe it's better to rely on docs than vscode suggestion, when applying such changes 🤔 I checked it and this is TypeScript feature.
>Parameters declaration with names starting with _ are exempt from the unused parameter checking. e.g.:
```
function returnNull(_a) { // OK
    return null;
}
```


More here: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html